autoload/outline.vim

```diff
@@ -234,7 +239,12 @@ export def OpenOutlineWindow()
 
   var prevWinID: number = win_getid()
 
-  :topleft :20vnew LSP-Outline
+  if opt.lspOptions.outlineOnRight
+    :setlocal splitright
```

Comment:
Instead of setting the 'splitright' option which may have unexpected side-effects,
can you use the "rightbelow' command modifier for the vnew?